project/dexter-server/swagger.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
This looks like an example file. Perhaps it should be deleted.

Reply:
You are right, I've removed it